test/integration/functional_test.go

```diff
@@ -124,7 +124,6 @@ func TestFunctional(t *testing.T) {
 			{"ServiceCmd", validateServiceCmd},
 			{"AddonsCmd", validateAddonsCmd},
 			{"PersistentVolumeClaim", validatePersistentVolumeClaim},
-			{"TunnelCmd", validateTunnelCmd},
```

Comment:
I think tunnel test should still be part of Functional Tests, since we consider this test as a Core Test that tests all functionalities of minikube.

when I meant to do it in serial, I meant tunnel test itself runs in parallel but all of it is own sub tests run in serial.

Reply:
@medyagh 
First time I tried the serial test into `validateTunnelCmd `, but the subtest failed because the process stopped like a deadlock.

After your this comment, I tried to add serial test into `validateTunnelCmd` of `TestFunctional` again.
And now serial test works and the `HyperKit_macOS` test all successed!!! 💯

https://storage.googleapis.com/minikube-builds/logs/7956/884f0e2/HyperKit_macOS.html#pass_TestFunctional%2fparallel%2fTunnelCmd%2fserial%2fAccessThroughDNS